perfrunbook/utilities/find_and_list_jar_with_so.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
```

Comment:
`set -e` given the number of `rm *` we don't want something failing and worst happening